server/routes/bitbucket/bitbucket_selection.py

```
@@ -196,6 +228,239 @@ def clear_workspace_selection(user_id):
+
+
+def cleanup_org_hooks(user_id: str, org_id: str, repo_full_names=None) -> None:
+    """Delete Aurora-created hooks + clear stored uuids for org repos.
+
+    ``repo_full_names=None`` means every Bitbucket repo in the org
+    (disconnect / clear-all); otherwise only the listed repos (deselect).
+    Best-effort: API failures still clear the stored uuid so the DB never
+    points at a hook we no longer manage.
+    """
+    with db_pool.get_admin_connection() as conn:
+        with conn.cursor() as cur:
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)